packages/lib/src/tabs/tabs.wc.svelte

```diff
@@ -0,0 +1,121 @@
+<svelte:options
+  customElement={{
+    tag: 'jp-tabs',
+    shadow: 'none'
+  }}
+/>
+
+<script lang="ts">
+  import { createEventDispatcher } from 'svelte';
+
+  export let tabs: string[] = ['tab 1', 'tab 2', 'tab 3'];
+  export let content: string[] = ['content 1', 'content 2', 'content 3'];
```

Comment:
Content should not be an attribute. You should be able to put content directly into `<jp-tabs>` and derive `tabs` from content attributes.
Demo would look something like this:
```
<jp-tabs>
    <div data-label="jedan">Content 1</div>
    <div data-label="dva">Content 2</div>
    <div data-label="tri">Content 3</div>
</jp-tabs>
```